src/s1reader/s1_reader.py

```diff
@@ -526,6 +527,45 @@ def get_track_burst_num(track_burst_num_file: str = esa_track_burst_id_file):
 
     return track_burst_num
 
+
+def get_anx_time_orbit(osv_list:list, anx_time_annotation: datetime.datetime, margin_anx_sec=60.0):
+    '''
+    Estimate the time of ascending node crossing (ANX) from orbit
+
+    parameters
+    ----------
+    osv_list: list
```

Comment:
```suggestion
    osv_list: list
```
Please, consider using a variable name that is easier to understand and read, e.g. 'state_vector_list'. Also, please describe briefly how each state vector should be describe. Is it a tuple/list of 2 or 3 elements?

Reply:
TODO: change the variable name throughout the code

Reply:
The variable name `osv_list` was changed to `orbit_state_vector_list` throughout the code.